src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateSDOCallbackHandler.java

```diff
@@ -666,6 +685,9 @@ private CallbackResponse prePopulateOrderDetailsPages(CallbackParams callbackPar
 
         updatedData.smallClaimsRoadTrafficAccident(tempSmallClaimsRoadTrafficAccident).build();
 
+        if (featureToggleService.isSdoR2Enabled()) {
+            populateDRHFields(callbackParams, updatedData, preferredCourt);
```

Comment:
Should we not check the DRH is selected along with sdo r2 flag

Reply:
> Should we not check the DRH is selected along with sdo r2 flag

No.. This is in about to start so we don't have if user is selected DRH or not.